common/lib/driver-definitions/api-report/driver-definitions.api.md

```diff
@@ -218,6 +219,14 @@ export interface IGenericNetworkError extends IDriverErrorBase {
     readonly statusCode?: number;
 }
 
+// @public (undocumented)
+export interface ILocationRedirectionError extends IDriverErrorBase {
+    // (undocumented)
+    readonly errorType: DriverErrorType.locationRedirection;
+    // (undocumented)
+    readonly patchedResolvedUrl: IResolvedUrl;
```

Comment:
maybe redirectedUrl? "resolved" word is present in type, and "patched" seems not be matching name of interrface.